### PR TITLE
Adapt ServicesManagerService #import and #export methods to new API

### PR DESCRIPTION
**Pull request by dgdavid**

### CODE REVIEW

File: src/modules/services_manager_service.rb [hunk 1]
```diff
@@ -150,48 +164,33 @@ def reset
...
-        when 'enable'
-          exists?(service.name) ? enable(service.name) : non_existent_services << service.name
-        when 'disable'
-          exists?(service.name) ? disable(service.name) : non_existent_services << service.name
-        else
-          log.error("Unknown status '#{service.status}' for service '#{service.name}'")
-        end
-      end
+      known_services = profile.services.select { |service| exists?(service.name) }
+      unknown_services = profile.services - known_services
```
Comment (jreidinger) [inline src/modules/services_manager_service.rb]: you can do it in one step: `known_services, unknown_service = profile.services.partition { |service| exists?(service.name) }`

Comment (dgdavid) [reply src/modules/services_manager_service.rb]: Nice! Thank you :)

File: src/modules/services_manager_service.rb [hunk 2]
```diff
@@ -150,48 +164,32 @@ def reset
...
+      log.info "Export: enabled services: #{enabled_services}, disabled services: #{disabled_services}"
 
-      {
-        'enable' => export_enable,
-        'disable' => export_disable,
-      }
+      { "enable" => enabled_services, "disable" => disabled_services }
     end
 
     def import(profile)
```
Comment (joseivanlopez) [inline src/modules/services_manager_service.rb]: NP: we can take the opportunity to add some documentation to this `#import` method.

**Comment by joseivanlopez** [comment]
As commented offline, codewise LGTM, only minor comments. Regarding unit tests for `#export`, we have to be sure that all cases are tested.

**Comment by dgdavid** [comment]
@joseivanlopez I just improved a little the test coverage. Do you thing that test are better now?

File: test/services_manager_service_test.rb [hunk 3]
```diff
@@ -292,66 +292,102 @@
...
+          expect(exported_services["enable"]).to include("dbus")
+        end
+      end
+
+      context "and is disabled" do
+        before do
+          allow(dbus).to receive(:start_mode).and_return(:manual)
+        end
+
+        it "does not export the service as enable" do
```
Comment (joseivanlopez) [inline test/services_manager_service_test.rb]: Typo: enable

File: test/services_manager_service_test.rb [hunk 4]
```diff
@@ -292,66 +292,102 @@
...
+        end
+
+        it "exports the service as enable" do
+          expect(exported_services["enable"]).to include("cups")
+          expect(exported_services["disable"]).to_not include("cups")
+        end
       end
 
-      context "but is proposed to be disabled" do
+      # FIXME this scenario should be fixed, see {#export} method
```
Comment (joseivanlopez) [inline test/services_manager_service_test.rb]: yardoc does not know how to resolve the link to  `{#export}` here. Use the full name instead: `{Yast::ServicesManagerServiceClass#export}`

File: src/modules/services_manager_service.rb [hunk 5]
```diff
@@ -166,6 +166,8 @@ def reset
 
     # Returns services to be exported to AutoYast profile
     #
+    # FIXME: should be checked (and decided wthat to do if so) if service is marked to be exportd as
```
[the remark this answers is not in the document]
Comment (dgdavid) [reply src/modules/services_manager_service.rb]: :see_no_evil:

File: test/services_manager_service_test.rb (hunk 4 above)
Comment (dgdavid) [reply test/services_manager_service_test.rb]: Please, do not pay attention to my previous comment. I just realized that the comment it is in the test file :see_no_evil:

**Pull request merged**